projects/storefrontlib/src/cms-components/myaccount/order/cancellations-returns/return-order-confirmation/return-order-confirmation.component.html

```diff
@@ -0,0 +1,14 @@
+<ng-container *ngIf="order$ | async as order">
+  <cx-cancellation-return-items
+    [entries]="returnedEntries"
+    [cancelOrder]="false"
+    [confirmRequest]="true"
+  >
+  </cx-cancellation-return-items>
+
```

Comment:
Just a thought, shouldn't we display the new total?

Reply:
we only display the returned total.